TSN/README.md

```diff
@@ -0,0 +1,28 @@
+# TSN
+
+This repo provides one method for Person Action Recognition task.
+
+## Get started
+Prerequisites
+pip3 install -r requirements.txt
+
+## Data preparation
+We use Kinetics400 to train and evaluate our models. Please refer to (https://github.com/open-mmlab/mmaction/tree/master/data_tools/kinetics400) for data prepration.
+download mini_dataset here for evaluate the model.
+```bash
+wget https://oneflow-public.oss-cn-beijing.aliyuncs.com/datasets/TSN/kinetics400.zip
+unzip kinetics400.zip
+```
+
+##Train a model
+To train a model, run sh train.sh
+
+Our model is trained using ResNet50 as backbones
+##Evaluate a model
+
+To evaluate a model, run sh infer.sh
+
```

Comment:
## Training
```bash
bash train.sh
```
Our model is trained using ResNet50 as backbones.

## Evaluating
```bash
bash infer.sh
```